fuzzbucket_client/__main__.py

```diff
@@ -1270,8 +1323,51 @@ def _with_ssh_opts(self, unknown_args: typing.List[str]) -> typing.List[str]:
             is None
         ):
             unknown_args = ["-o", "UserKnownHostsFile=/dev/null"] + unknown_args
+
+        if ssm:
+            return self._with_ssh_ssm_proxy_command(box, unknown_args)
+
         return unknown_args
 
+    def _with_ssh_ssm_proxy_command(
+        self, box: Box, unknown_args: typing.List[str]
+    ) -> typing.List[str]:
+        unknown_args_string = " ".join(unknown_args)
+        if (
+            re.search(" -o ProxyCommand=.+", unknown_args_string, re.IGNORECASE)
+            is not None
+        ):
+            return unknown_args
+
+        aws_region = box.get(
+            "region",
+            self._env.get(
+                "AWS_REGION",
+                self._env.get("AWS_DEFAULT_REGION", "us-east-2"),
```

Comment:
This is pretty buried as a fallback. Should it default to the region FB is deployed to?
Can FB spin up instances in regions other than where it is deployed?

Reply:
My thinking was that a given human may be using a `fuzzbucket-client` installation to talk to more than one `fuzzbucket` deployment (I do this ✋🏼) living in different regions, and the local environment `AWS_REGION` and `AWS_DEFAULT_REGION` should have lower precedence than whatever region reported by the `fuzzbucket` deployment (via the `box` API type).

Reply:
Said another way, if you want the first region selected to be the same as the region where the `fuzzbucket` server-side is deployed, this is the intended behavior 👍🏼